app/views/inventories/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% if can? :read, @inventory%>
+<div class="show-inventory max-w-3xl mx-auto mt-8 bg-white p-4 rounded-lg shadow-md">
+  <h2 class="text-2xl font-semibold text-blue-500 mb-4"><%= @inventory.name %></h2>
+  <div class="food-add">
+    <%= link_to "Add Inventory Food", new_inventory_path(@inventory), class: "bg-blue-500 text-white px-4 py-2 rounded hover:bg-blue-600"%>
```

Comment:
- You have done well so far, but when I click on the `Add inventory food` button, it should take me to a page where I can from a list of loaded foods, not to a form where I add a new inventory, please fix this 🙏